x-pack/solutions/observability/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -291,7 +291,7 @@ function useTableData({
   serviceName: string;
   start: string;
   transactionType: string | undefined;
-  renderedItemIndices: VisibleItemsStartEnd;
+  renderedItems: ApiResponse['transactionGroups'];
```

Comment:
Q: Just curious what happened that it was broken with the previous implementation, and it worked also before the mentioned version - it is the API response that changed?